lib/ocpp/common/websocket/websocket_libwebsockets.cpp

```diff
@@ -715,7 +716,7 @@ void WebsocketTlsTPM::close(websocketpp::close::status::value code, const std::s
     // Notify any message senders that are waiting, since we can't send messages any more
     msg_send_cv.notify_all();
 
-    std::thread closing([this]() { this->closed_callback(websocketpp::close::status::normal); });
+    std::thread closing([this]() { this->closed_callback(WebsocketCloseReason::Normal); });
```

Comment:
Shouldnt this use the `code` argument?

Reply:
This works on a fire-and-forget mode. We just notice the thread that we want to close, and the thread will do the close as soon as possible when it sees fit. We are not waiting for the close in order to get a response. Is there any case in which we care how/when we close? Additional mutex/conditional variables will have to be added in order to block the calling thread until the close is done if required.